pkg/scheduler/scheduler.go

```diff
@@ -275,65 +268,43 @@ func (s *Scheduler) worker() {
 
 func (s *Scheduler) getPlacement(resourceBinding *workv1alpha2.ResourceBinding) (policyv1alpha1.Placement, string, error) {
 	var placement policyv1alpha1.Placement
-	var clusterPolicyName string
-	var policyName string
-	var policyNamespace string
 	var err error
-	if clusterPolicyName = util.GetLabelValue(resourceBinding.Labels, policyv1alpha1.ClusterPropagationPolicyLabel); clusterPolicyName != "" {
-		var clusterPolicy *policyv1alpha1.ClusterPropagationPolicy
-		clusterPolicy, err = s.clusterPolicyLister.Get(clusterPolicyName)
-		if err != nil {
-			return placement, "", err
-		}
-
-		placement = clusterPolicy.Spec.Placement
-	}
 
-	if policyName = util.GetLabelValue(resourceBinding.Labels, policyv1alpha1.PropagationPolicyNameLabel); policyName != "" {
-		policyNamespace = util.GetLabelValue(resourceBinding.Labels, policyv1alpha1.PropagationPolicyNamespaceLabel)
-		var policy *policyv1alpha1.PropagationPolicy
-		policy, err = s.policyLister.PropagationPolicies(policyNamespace).Get(policyName)
-		if err != nil {
-			return placement, "", err
-		}
-
-		placement = policy.Spec.Placement
+	placementPtr := resourceBinding.Spec.Placement
+	if placementPtr == nil {
```

Comment:
```
func (s *Scheduler) resourceBindingEventFilter(obj interface{}) bool {
	accessor, err := meta.Accessor(obj)
	if err != nil {
		return false
	}

	switch t := obj.(type) {
	case *workv1alpha2.ResourceBinding:
		if !schedulerNameFilter(s.schedulerName, t.Spec.SchedulerName) {
			return false
		}
	case *workv1alpha2.ClusterResourceBinding:
		if !schedulerNameFilter(s.schedulerName, t.Spec.SchedulerName) {
			return false
		}
	}

	return util.GetLabelValue(accessor.GetLabels(), policyv1alpha1.PropagationPolicyNameLabel) != "" ||
		util.GetLabelValue(accessor.GetLabels(), policyv1alpha1.ClusterPropagationPolicyLabel) != ""
}
```
In fact, placement == nil has two situations: (1). No policy matched. It will be filtered because it has no label. (2) Users delete it from rb. I think it's an uncorrect behavior. I prefer the current logic because it will occur a error log indicating users.